doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -84,37 +174,261 @@ writeTextFile {
   executable = true;
   destination = "/bin/my-file";
 }
-# Writes contents of file to /nix/store/<store path>
+```
+:::
+
```

Comment:
Please make these into separate examples. Also, pull the prose out into the surrounding text.

Reply:
This has been done.